jwt/jwt_helpers.c

```diff
@@ -124,5 +124,79 @@ NN_DIGIT * private_key_from_pem(const char *pem_string)
     EC_KEY_free(ec_key);
     BIO_free(bio);
 
+    return key;
+}
+
+point_t * read_public_key(const char *pem_file)
```

Comment:
Do not duplicate code.

If your intention is to add a function that takes the file name instead of the key itself, write a function that reads the content of the file and then call  already existing `public_key_from_pem.`